config/locales/en.yml

```diff
@@ -528,7 +528,7 @@ en:
       schools_on_boarding_candidate_requirements_choice:
         has_requirements: "Do you have any candidate requirements?"
       schools_on_boarding_dbs_requirement:
-        dbs_policy_conditions: 'Do you require candidates to have or get a DBS check?'
+        dbs_policy_conditions: 'Do candidates need to have or get a DBS check?'
```

Comment:
Really, we should remove the top part of this page (see image) and make the 'Do candidates need to have or get a DBS check?' question the heading 1. I'm not sure if that affects any other content (ie if this page renders other content under different circumstances).

![Screenshot 2021-12-06 at 16 54 01](https://user-images.githubusercontent.com/85290597/144888346-fd38071e-444b-44c8-82b6-1845b3363e2d.png)